ci-operator/step-registry/baremetal/lab/ipi/conf/network/vips/baremetal-lab-ipi-conf-network-vips-commands.sh

```
@@ -5,6 +5,18 @@ set -o errexit
 
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=ERROR
+  -i "${CLUSTER_PROFILE_DIR}/ssh-key")
+
+CLUSTER_NAME="$(<"${SHARED_DIR}/cluster_name")"
+
+ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" \
+  "systemd-cat -t '${CLUSTER_NAME}' -p5 echo 'baremetal-lab-ipi-conf-network-vips: Configuring VIPs'"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)